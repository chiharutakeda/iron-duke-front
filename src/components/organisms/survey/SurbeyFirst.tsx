import React from 'react';
import { useRouter } from 'next/router';
const SurbeyFirst = () => {
  const router = useRouter();
  return (
    <>
      <div>SurbeyFirst</div>
      <button
        onClick={() => {
          router.push('SurbeySecond');
        }}
      >
        次へ
      </button>
    </>
  );
};

export default SurbeyFirst;
