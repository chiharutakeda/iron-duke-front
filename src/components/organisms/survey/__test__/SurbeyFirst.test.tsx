import ReactTestRenderer from 'react-test-renderer';
import SurbeyFirst from '../SurbeyFirst';

it('SurbeyFirst', async () => {
  const tree = await ReactTestRenderer.create(<SurbeyFirst />).toJSON();
  await expect(tree).toMatchSnapshot();
});
