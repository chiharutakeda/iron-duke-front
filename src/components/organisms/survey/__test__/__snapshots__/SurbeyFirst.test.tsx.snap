// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurbeyFirst 1`] = `
Array [
  <div>
    SurbeyFirst
  </div>,
  <button
    onClick={[Function]}
  >
    次へ
  </button>,
]
`;
